### PR TITLE
UI & Routing : playing with django and its templates draft

### PR DESCRIPTION
**Pull request by shlior7**
This PR, is just a draft for playing with the functionality of django and its templates for our global page
eventually this will be split to several PR's.

### CODE REVIEW

**Comment by shlior7** [comment]
image

table connected to all houses 
chart connected to all expenses

**Comment by svatares** [comment]
@shlior7 I know it marked as "draft", but giving you a heads-up, commit/PR should be encapsulated and relevant only for one thing/bug/feature/change.
By the title of this PR I can understand that it is not relevant only for one thing.
Please think if you need to split it to several PRs.
(I haven't reviewed the files since it is marked as "draft")

**Comment by shlior7** [comment]
image

working filter with either parent profession checking

File: expenses/migrations/0001_initial.py [hunk 1]
```diff
@@ -21,9 +19,34 @@ class Migration(migrations.Migration):
             fields=[
                 ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                 ('description', models.TextField(default='', max_length=250)),
-                ('category', models.CharField(choices=[('Rent', 'Rent'), ('Mortgage', 'Mortgage'), ('Bills', 'Bills'), ('Transportation', 'Transportation'), ('Clothing', 'Clothing'), ('Healthcare', 'Healthcare'), ('Food', 'Food'), ('Insurance', 'Insurance'), ('Kids', 'Kids'), ('Culture', 'Culture'), ('Vacations', 'Vacations'), ('Other', 'Other')], default='Other', max_length=32)),
-                ('amount', models.IntegerField(blank=True, null=True, validators=[django.core.validators.MinValueValidator(0)])),
-                ('date', models.DateTimeField(verbose_name=datetime.datetime(2022, 4, 6, 20, 39, 30, 83754, tzinfo=utc))),
+                (
```
Comment (ronyyosef) [inline expenses/migrations/0001_initial.py:22]: I have fixed the flake8, you can leave the long lines here

File: house/migrations/0001_initial.py [hunk 2]
```diff
@@ -34,31 +33,69 @@ class Migration(migrations.Migration):
...
-                                                                  ('Vet', 'Vet'), ('Nurse', 'Nurse'),
-                                                                  ('Technichian', 'Technichian'),
-                                                                  ('Cleaner', 'Cleaner'), ('Other', 'Other'),
-                                                                  ('Unemployed', 'Unemployed')],
-                                                         default='Other', max_length=50)),
-                ('income', models.IntegerField(
-                    blank=True, null=True, validators=[django.core.validators.MinValueValidator(0)])),
-                ('children', models.IntegerField(
-                    blank=True, null=True, validators=[django.core.validators.MinValueValidator(0)])),
+                (
```
Comment (ronyyosef) [inline house/migrations/0001_initial.py:36]: same here about flake8

**Review by ronyyosef — COMMENTED** [review]
We need to have a short meeting about this code as you wrote in our slack team
overall looks great

**Comment by adiabramovitch** [comment]
@shlior7 How do we proceed with this PR?

**Pull request closed without merging**